pkg/resources/marketplace/manager.go

```diff
@@ -88,14 +92,25 @@ func (m *Manager) InstallOperator(ctx context.Context, serverClient k8sclient.Cl
 			TargetNamespaces: operatorGroupNamespaces,
 		},
 	}
+	//get the rhmi CR to check if its a sandbox install
+	sandboxRhmiCR := &integreatlyv1alpha1.RHMI{}
+
+	// don't like hard coding but can't see another way to get the rhoam sandbox CR at this point
+	err = serverClient.Get(ctx, k8sclient.ObjectKey{Namespace: "sandbox-rhoam-operator", Name: "rhoam"}, sandboxRhmiCR)
+	if err != nil {
+		fmt.Sprintf("failed marketplace manager to get sandbox rhmi CR  in namespace rhoam")
+	}
+
 	//TODO can maybe return this to a create only function after all clusters are updated in production
 	//err = serverClient.Create(ctx, og)
 	_, err = controllerutil.CreateOrUpdate(ctx, serverClient, og, func() error {
 		og.Namespace = t.Namespace
 		og.Name = OperatorGroupName
 		og.Labels = map[string]string{"integreatly": t.SubscriptionName}
 		if csv.Status.Phase == "Succeeded" && csv.Status.Reason == "InstallSucceeded" {
-			og.Spec.TargetNamespaces = operatorGroupNamespaces
+			if !integreatlyv1alpha1.IsRHOAMMultitenant(integreatlyv1alpha1.InstallationType(sandboxRhmiCR.Spec.Type)) {
```

Comment:
are we not going to have a nil pointer here at some point?
If the RHMI CR has not been found, we are sending in empty rhmi CR?